tests/benchmarks/test_spill.py

```diff
@@ -0,0 +1,90 @@
+import uuid
+
+import dask.array as da
+import numpy as np
+import pytest
+from coiled import Cluster
+from dask.distributed import Client, wait
+from toolz import merge
+
+from ..utils_test import (
+    cluster_memory,
+    print_size_info,
+    scaled_array_shape,
+    scaled_array_shape_quadratic,
+)
+
+
+@pytest.fixture(scope="module")
+def spill_cluster(dask_env_variables, cluster_kwargs, gitlab_cluster_tags):
+    with Cluster(
+        name=f"spill-{uuid.uuid4().hex[:8]}",
+        environ=merge(
+            dask_env_variables,
+            {
+                # Ensure that no tasks are not retried on worker ungraceful termination
+                # caused by out-of-memory issues
+                "DASK_DISTRIBUTED__SCHEDULER__ALLOWED_FAILURES": "0",
+            },
+        ),
+        tags=gitlab_cluster_tags,
+        **cluster_kwargs["spill_cluster"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture
+def spill_client(spill_cluster, cluster_kwargs, upload_cluster_dump, benchmark_all):
+    n_workers = cluster_kwargs["spill_cluster"]["n_workers"]
+    with Client(spill_cluster) as client:
+        spill_cluster.scale(n_workers)
+        client.wait_for_workers(n_workers)
+        client.restart()
+        with upload_cluster_dump(client), benchmark_all(client):
+            yield client
+
+
+@pytest.mark.parametrize(
+    "keep_around", [pytest.param(False, id="release"), pytest.param(True, id="keep")]
+)
+@pytest.mark.parametrize(
+    "compressible",
+    [pytest.param(False, id="uncompressible"), pytest.param(True, id="compressible")],
+)
+def test_spilling(spill_client, compressible, keep_around):
+    memory = cluster_memory(spill_client)  # 36 GiB
+    shape = scaled_array_shape(memory * 1.79, ("x", "x"))  # 64 GiB
```

Comment:
`memory * 1.79` is oddly specific yet not understandable without the comment. Why not simply use a constant? Is this to enable benchmarking larger clusters with linearly scaling data sizes?

Reply:
It is to match the previous implementation, so that we don't need to declare bankruptcy on the historical measures.